esy/Package.ml

```diff
@@ -53,6 +53,34 @@ module CommandList = struct
 
 end
 
+(**
+ * Scripts with keys as specified in "scripts".
+ *)
+module Scripts = struct
+
+  type script = {
+    key : string;
+    command : string;
```

Comment:
Let's make it `command: Cmd.t` (I think it is done for `esy.build` and `esy.install` already.

Reply:
I think we can add new `Json.Parse.cmd` combinator which will parse either a string of an array of strings into a `Cmd.t`.